protocol/0028-GOVE-governance.md

```diff
@@ -453,11 +444,9 @@ Below `*` stands for any of `asset, market, updateMarket, updateNetParam, freeFo
 
 
 ### Freeform governance proposals
-- [ ] A freeform governance proposal with a description field that is empty, or not between 0 and 255 characters, will be rejected (<a name="0028-GOVE-019" href="#0028-GOVE-019">0028-GOVE-019</a>)
-- [ ] A freeform governance must contain a hash field and it must not be null, but no other check is done to verify it (<a name="0028-GOVE-020" href="#0028-GOVE-020">0028-GOVE-020</a>)
```

Comment:
Could we have this AC replaced with one ensuring that the Title is specified and is < 100 chars?

Reply:
done in https://github.com/vegaprotocol/specs/pull/1186/commits/0a9f0f2a1ce1de3301dafba2bd1496fa38166de2